packages/core/src/components/ui/ReviewsAndRatings/ReviewsAndRatings.tsx

```diff
@@ -1,11 +1,27 @@
+import { RatingSummary } from '@faststore/ui'
+import { usePDP } from 'src/sdk/overrides/PageProvider'
+import useScreenResize from 'src/sdk/ui/useScreenResize'
+
 export type ReviewsAndRatingsProps = {
   title: string
 }
 
 function ReviewsAndRatings({ title, ...otherProps }: ReviewsAndRatingsProps) {
+  const context = usePDP()
+  const { isDesktop } = useScreenResize()
+
+  const {
+    product: { rating },
+  } = context?.data
```

Comment:
suggestion: we could be more cautious here since data and product may not exist on PDPContext
```suggestion
  const rating = context?.data?.product?.rating
```